README-emscripten.md

```diff
@@ -0,0 +1,89 @@
+# Building QPDF with Emscripten
+
+This guide explains how to compile QPDF to JavaScript/WebAssembly using Emscripten.
+
+> **Note**: For pre-built bundles and a convenient wrapper around the WebAssembly build, check out [qpdf.js](https://github.com/j3k0/qpdf.js).
+
+## Prerequisites
+
+Install Emscripten by following the [official installation instructions](https://emscripten.org/docs/getting_started/downloads.html).
+
+## Quick Start
+
+1. Create and enter build directory:
+```bash
+mkdir build-emscripten && cd build-emscripten
+```
+
+2. Build and copy to examples:
+```bash
+# Configure and build
+emcmake cmake .. \
```

Comment:
This didn't work for me. I got errors about zlib, openssl, and jpeg, even though your wasm flags seem to not require them. I'm not sure why...a little quick tinkering didn't reveal it.